### PR TITLE
Escaped the attributes before echo

### LINKED ISSUE #1564: Escaping attributes value before echo
**Describe the bug**
HTML is getting break if in the attribute we pass any special character 
https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fp137.p2.n0.cdn.getcloudapp.com%2Fitems%2F8LuPvq1l%2FImage%25202020-09-21%2520at%252010.55.05%2520pm.png

Line Number:
plugins/buddyboss-platform/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php Line number 10
plugins/buddyboss-platform/bp-templates/bp-nouveau/buddypress/search/loop/post.php Line number 8

Right now:
```
<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php the_title(); ?>"/>
```

It should be:
```
<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php esc_attr_e( get_the_title() ); ?>"/>
```

**Expected behavior**
Always add esc_attr before passing

**Screenshots**
https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fp137.p2.n0.cdn.getcloudapp.com%2Fitems%2F8LuPvq1l%2FImage%25202020-09-21%2520at%252010.55.05%2520pm.png

### PR DESCRIPTION
**Pull request by ngolwalkar**
### Changes proposed in this Pull Request:

Fixes #1564

### How to test the changes in this Pull Request:

1. Add a new post with a title which have special characters in it.
2. Go to the frontend and search for the same post you created.
3. Now it won't break HTML.

### Proof Screenshots or Video

https://prnt.sc/12ff93n

### CODE REVIEW

File: src/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 			<img
 				src="<?php echo $result['post_thumbnail']; ?>"
 				class="attachment-post-thumbnail size-post-thumbnail wp-post-image"
-				alt="<?php the_title() ?>"
+				alt="<?php esc_attr( the_title() ); ?>"
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php:10]: @ngolwalkar `the_title()` will print directly so `esc_attr` will not work. You should use `echo esc_attr( get_the_title() );` 

also `the_title()` no need to escape as is already escaping by WordPress

File: src/bp-templates/bp-nouveau/buddypress/search/loop/post.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<div class="list-wrap">
 		<div class="item-avatar">
 			<a href="<?php the_permalink(); ?>">
-				<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php the_title() ?>"/>
+				<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php esc_attr( the_title() ); ?>"/>
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/post.php:8]: @ngolwalkar Needs to make the same changes as above.

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
See here for more info https://wordpress.stackexchange.com/questions/52597/should-i-escape-wordpress-functions-like-the-title-the-excerpt-the-content/52600#52600

File: src/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php [hunk 3]
```diff
@@ -7,7 +7,7 @@
 			<img
 				src="<?php echo $result['post_thumbnail']; ?>"
 				class="attachment-post-thumbnail size-post-thumbnail wp-post-image"
-				alt="<?php the_title() ?>"
+				alt="<?php esc_attr( get_the_title() ); ?>"
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php:10]:
```suggestion
				alt="<?php echo esc_attr( get_the_title() ); ?>"
```

File: src/bp-templates/bp-nouveau/buddypress/search/loop/post.php (hunk 2 above)
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/post.php:8]:
```suggestion
				<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php echo esc_attr( get_the_title() ); ?>"/>
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need changes

**Review by chetansatasiya — APPROVED** [review]
LGTM 🚀 

@ngolwalkar Merge in dev and put in testing

**Comment by sumitbuddyboss** [comment]
Hello @ngolwalkar 

I can't reproduce this issue in my local site and demo site as well, for the prof please check the below screenshots.

**Demo site**
https://prnt.sc/1o0f2wu

**Local site**
https://prnt.sc/1o0fboc

@ngolwalkar Please make sure from your end.

cc @chetansatasiya 

Thank you

File: src/bp-templates/bp-nouveau/buddypress/search/loop/post.php [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	<div class="list-wrap">
 		<div class="item-avatar">
 			<a href="<?php the_permalink(); ?>">
-				<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php the_title() ?>"/>
+				<img src="<?php echo $result['post_thumbnail']; ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php echo esc_attr( get_the_title() ); ?>"/>
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/post.php:8]:
```suggestion
				<img src="<?php echo esc_url( $result['post_thumbnail'] ); ?>" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="<?php echo esc_attr( get_the_title() ); ?>"/>
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@ngolwalkar Add the `esc_url` in image src.

**Pull request merged**